src/hooks/navHooks/useUpdateNavHook.ts

```diff
@@ -39,13 +43,13 @@ export const useUpdateNavHook = (
     },
     {
       onSuccess: () => {
-        queryClient.invalidateQueries([GET_NAV_KEY, siteName])
+        queryClient.invalidateQueries([NAVIGATION_CONTENT_KEY, siteName])
```

Comment:
We're not required to update second nav contents cos only the title may change right?

Reply:
Hmm we still need to invalidate this key because the user may have made changes to the structure of the navbar data object. Invalidating here ensures the user gets the latest SHA from the backend after the change was made.

Reply:
not this key, i meant `DIR_SECOND_LEVEL_CONTENT_KEY`